concent_api/core/views.py

```diff
@@ -586,3 +719,20 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
         return False
     else:
         raise exceptions.UnexpectedResponse()
+
+
+def decode_key(key: str) -> bytes:
+    return b64decode(key.encode('ascii'), validate = True)
+
+
+def decode_client_public_key(request):
+    return decode_key(request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'])
```

Comment:
You should add a comment or an `assert` to explain why you assume that the header is always present because from the declaration alone it's not obvious that you only use it in views that use `api_view` decorator.

Reply:
:+1: